{{cookiecutter.project_slug}}/{{cookiecutter.module_slug}}/_cli.py

```diff
@@ -1,6 +1,8 @@
-"""
-The `{{ cookiecutter.project_slug }}` entrypoint.
-"""
+"""The `{{ cookiecutter.project_slug }}` entrypoint."""
+
+# ruff: noqa: T201
+# T201 `print` found
```

Comment:
Similar thought here: maybe we just exclude this rule from `_cli.py`, since it's somewhat reasonable for a CLI module to call `print(...)`?